aci/resource_aci_fabricnodeidentp.go

```diff
@@ -74,6 +75,9 @@ func resourceAciFabricNodeMember() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					"unspecified",
 					"remote-leaf-wan",
+					"tier-2-leaf",
+					"virtual",
+					"leaf",
```

Comment:
I don't find leaf in meta why is it here?
Latest meta also allows border-gateway

Reply:
fixed